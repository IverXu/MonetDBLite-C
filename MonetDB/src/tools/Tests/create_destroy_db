#!/bin/sh

Mlog -x $MCREATEDB -db TestDB1
Mlog   "echo 'quit;' | $MSERVER -db TestDB1"
        echo 'quit;' | $MSERVER -db TestDB1
echo
Mlog -x $MDESTROYDB -db TestDB1

TSTDBHOME=$MONETFARM
unset MONETFARM

Mlog -x $MCREATEDB -home $TSTDBHOME -db TestDB2
Mlog   "echo 'quit;' | $MSERVER -home $TSTDBHOME -db TestDB2"
        echo 'quit;' | $MSERVER -home $TSTDBHOME -db TestDB2
echo
Mlog -x $MDESTROYDB -home $TSTDBHOME -db TestDB2

