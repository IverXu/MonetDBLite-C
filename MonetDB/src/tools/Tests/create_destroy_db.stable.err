stderr of test 'create_destroy_db` in directory 'src/tools` itself:

#> Mtest:	other create_destroy_db
#> Mtest:	./create_destroy_db create_destroy_db
#> Mlog:	Mcreatedb -user manegold -db TestDB1
Mcreatedb: using default dbdir /dbfarm/Monet/StM/Current/monet/IRIX646.5/dbfarm/TestDB1.
#> Mlog:	echo 'quit;' | Mtimeout -timeout 99 Mserver -yaccsilent -d8 -db TestDB1
#> Mlog:	Mdestroydb -db TestDB1
#> Mlog:	Mcreatedb -user manegold -home /dbfarm/Monet/StM/Current/monet/IRIX646.5 -db TestDB2
Mcreatedb: using default dbdir /dbfarm/Monet/StM/Current/monet/IRIX646.5/dbfarm/TestDB2.
#> Mlog:	echo 'quit;' | Mtimeout -timeout 99 Mserver -yaccsilent -d8 -home /dbfarm/Monet/StM/Current/monet/IRIX646.5 -db TestDB2
#> Mlog:	Mdestroydb -home /dbfarm/Monet/StM/Current/monet/IRIX646.5 -db TestDB2
