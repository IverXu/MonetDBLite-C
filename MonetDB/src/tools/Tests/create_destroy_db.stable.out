stdout of test 'create_destroy_db` in directory 'src/tools` itself:

#> Mtest:	other create_destroy_db
#> Mtest:	Mtimeout -timeout 1890 ./create_destroy_db create_destroy_db
#> Mlog:	Mtimeout -timeout 210 Mcreatedb -user monet -db TestDB1
Mcreatedb: processing database 'TestDB1' in /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1.
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1/bat 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1/users 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1/users/monet 
Mcreatedb: calling 'Mserver -db TestDB1' for the first time..
Mcreatedb: database 'TestDB1' has been initialized succesfully!
#> Mlog:	echo 'quit;' | Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db TestDB1
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
>
#> Mlog:	Mtimeout -timeout 210 Mdestroydb -db TestDB1
Mdestroydb: processing database 'TestDB1' in /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB1.
#> Mlog:	Mtimeout -timeout 210 Mcreatedb -user monet -home /var/tmp/monet.GNU_32/monet/IRIX646.5 -db TestDB2
Mcreatedb: processing database 'TestDB2' in /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2.
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2/bat 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2/users 
# /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2/users/monet 
Mcreatedb: calling 'Mserver -db TestDB2' for the first time..
Mcreatedb: database 'TestDB2' has been initialized succesfully!
#> Mlog:	echo 'quit;' | Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -home /var/tmp/monet.GNU_32/monet/IRIX646.5 -db TestDB2
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
>
#> Mlog:	Mtimeout -timeout 210 Mdestroydb -home /var/tmp/monet.GNU_32/monet/IRIX646.5 -db TestDB2
Mdestroydb: processing database 'TestDB2' in /var/tmp/monet.GNU_32/monet/IRIX646.5/dbfarm/TestDB2.
